test.py

```diff
@@ -107,10 +107,10 @@ def download_test(self, filename, recursive, destination=None,
         os.mkdir(temp_in)
         os.chdir(temp_in)
         try:
-            with SCPClient(self.ssh.get_transport()) as scp:
-                scp.get(filename,
-                        destination if destination is not None else u'.',
-                        preserve_times=True, recursive=recursive)
+            get(self.ssh.get_transport(),
```

Comment:
The method versions of get and put are also used in the tests. I didn't want to double all the tests but somehow still cover both codepaths.